cayennelpp/lpp_type.py

```diff
@@ -520,4 +525,7 @@ def get_lpp_type(tid):
     """Returns the LppType instance for a given `tid` or `None` if not found"""
     if not isinstance(tid, int):
         raise AssertionError()
-    return next(filter(lambda x: x.tid == tid, LPP_TYPES), None)
+    try:
+        return next(filter(lambda x: x.tid == tid, LPP_TYPES))
```

Comment:
mhm, no lambdas in micropython? than we might need a different thing here?

Reply:
Lambdas works, but the *fallback* aka *default* value is not supported.

Reply:
ah now I get it, so the problem is with the `next`?

Reply:
ah [here](https://docs.micropython.org/en/latest/genrst/modules.html#second-argument-to-next-is-not-implemented) it is, okay: understood. Please add a comment that this workaround is required for micropython to explain why we not use the second argument here.

Reply:
and btw. in your (longer) commit message you state the problem is with filter, but (see reference above) it is actually with `next`, please rephrase accordingly, THX!